### DISCUSSION TITLE
突然请求cleancloud报500

### DISCUSSION THREAD

**Discussion by gumpBOBO** [walinejs/waline#1211] [Q&A] [answered]
{errno: 500, errmsg: "undefined [404 GET server.ganxb2.com/1.1/classes/Comment]"}
errmsg: "undefined [404 GET server.ganxb2.com/1.1/classes/Comment]"
errno: 500

cleanclooud我是国内，前面都是好好的，waline版本是2.5.1

**Comment by lizheming** [comment]
https://server.ganxb2.com LeanCloud 绑定域名需要备案

![image](https://user-images.githubusercontent.com/424491/180367178-e3129241-a9b1-40b4-8b6a-34683bf342d6.png)

**Reply by gumpBOBO** [reply]
请问下你怎么访问出现这个的，我直接开这个就进了leancloud的控制面板去了

**Comment by gumpBOBO** [comment]
追加一下控制面板报这个跨域，我要在环境变量里+东西吗？
Access to fetch at 'https://waline-vercel-ecru.vercel.app/comment?path=%2Fposts%2Fbdfafec0.html&pageSize=10&page=1&lang=zh-CN' from origin 'https://blog.ganxb2.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

**Reply by lizheming** [reply]
https://github.com/walinejs/waline/issues/542 多搜索一下

**Reply by gumpBOBO** [reply]
好像不是这个问题。。。 那个问题是双斜线//comment...
leancloud的安全域名博客和评论我也都+了
我改了配置又重新发布了还是不行
辛苦大大了。。我再去看看，感觉是leancloud是不是在搞事情
最郁闷的是我这个星期都没动博客代码。突然就挂了。。前面都是好的

**Reply by gumpBOBO** [reply]
最后我换成了国际版可行。。楼下的方法我试了好像也不行。。不知道问题在哪~~感谢大大

> #542 多搜索一下

**Comment by akatsukiro** [comment, marked as answer]
我也出现了相同的问题，报错一致，但是我的域名是有备案的，请问是什么原因呢？

**Reply by akatsukiro** [reply]
`LEAN_SERVER` 加 `https://` 即可解决

**Reply by gumpBOBO** [reply]
啊啊啊，这样，我换成国际版了。。也ok，免得出问题，不过也算是有了另外的解决办法
